config_src/drivers/nuopc_cap/mom_surface_forcing_nuopc.F90

```diff
@@ -891,8 +899,10 @@ subroutine convert_IOB_to_forces(IOB, forces, index_bounds, Time, G, US, CS)
                  G%mask2dCv(i,J)*forces%tauy(i,J)**2) / (G%mask2dCv(i,J-1) + G%mask2dCv(i,J))
 
       if (CS%read_gust_2d) then
+        forces%tau_mag(i,j) = CS%gust(i,j) + sqrt(taux2 + tauy2)
         forces%ustar(i,j) = sqrt(CS%gust(i,j)*Irho0 + Irho0*sqrt(taux2 + tauy2))
       else
+        forces%tau_mag(i,j) = CS%gust(i,j) + sqrt(taux2 + tauy2)
```

Comment:
should this be forces%tau_mag(i,j) = CS%gust(i,j)_const + ....... ?

Reply:
Yes, @jiandewang, you are indeed correct.  This line should be `forces%tau_mag(i,j) = CS%gust_const + sqrt(taux2 + tauy2)`.  Sorry about this oversight.

Reply:
just got confirmation from Bin Li: HAFS-MOM6 works fine now with the correction on the typo.

Reply:
The corrected expression has now been merged into the PR.